count.py

```diff
@@ -40,10 +52,20 @@ def count(file, column, limit, interval, match, update_minutes, outfile):
                     dtime = datetime.strptime(row[indices[1]][0:-9], '%Y/%m/%d %H:%M:%S')
                     if dtime.hour < int(interval[0]) or dtime.hour > int(interval[1]):
                         continue
-                count_map[row[index]] = count_map.get(row[index], 0) + 1
 
-        alert_list = sorted(count_map.items(),
-                            key = lambda kv:(kv[1], kv[0]), 
+                current_count = count_map.get(row[index], 0)
+                if current_count != 0:
+                    current_count = current_count.get('count')
+                count_map[row[index]] = {"count": (current_count + 1), "fuzzy": False}
```

Comment:
If we don't use fuzzy matching and this dict key is set to False by default could we keep the same `column: count` formatting?

Reply:
sure thing 👍

Reply:
I decided to keep this here, but I remove it before writing to csv; otherwise it got a little messy if there was a fuzzy matching request...